### PR TITLE
Map system toast and file dragging handler in Screen, rename misnamed Screen

### PR DESCRIPTION
**Pull request by LambdAurora**
Changelog:
- `GraphicsConfirmationScreen` -> `DialogScreen`
  - I believe to have misnamed this screen as its purpose seems more wide even if only used for graphics. Could be used for other stuff in theory.
  - `field_25678` -> `linesY`
  - `field_25679` -> `buttonWidth`
- Screen: `method_29638` -> `filesDragged`
  - I followed the same naming scheme as `keyReleased`, `keyPressed`, `mouseClicked`, etc.
- `SystemToast` maps:
  - `field_25037` -> `lines`
  - `field_25038` -> `width`
  - `method_29046` -> `drawPart` I'm not too sure about that name, open to suggestions
  - `method_29047` -> `create`
  - `method_29627` -> `addPackCopyFailure`
- `Toast` maps:
  - `method_29049` -> `getWidth`
  - `method_29050` -> `getHeight`

### CODE REVIEW

File: mappings/net/minecraft/client/toast/SystemToast.mapping
Comment (liach) [inline mappings/net/minecraft/client/toast/SystemToast.mapping]: `create`? `copy`?

Comment (LambdAurora) [reply mappings/net/minecraft/client/toast/SystemToast.mapping]: more `create` than `copy`, it just ensures that while giving you a new instance of SystemToast it has the right width.

Comment (liach) [reply mappings/net/minecraft/client/toast/SystemToast.mapping]: is this an instance method or a static method? Should rename to `create` if this is just a factory static method.

Comment (LambdAurora) [reply mappings/net/minecraft/client/toast/SystemToast.mapping]: It's a static method

Comment (liach) [reply mappings/net/minecraft/client/toast/SystemToast.mapping]:
```suggestion
	METHOD method_29047 create (Lnet/minecraft/class_310;Lnet/minecraft/class_370$class_371;Lnet/minecraft/class_2561;Lnet/minecraft/class_2561;)Lnet/minecraft/class_370;
```

**Review by liach — APPROVED** [review]

**Review by shedaniel — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Comment by liach** [comment]
Should be good to merge once this is merging into 28a

**Pull request merged**
